### PR TITLE
Test against Kafka 1.1.1

### PR DESCRIPTION
**Pull request by twm**
- [x] Add configuration files for 1.1.1
- [x] Update `_KafkaFixture`
- [x] Add 1.1.1 to Travis CI
- [x] Update tests to run on 1.1.1
  - [x] `TestAfkakClientIntegration.test_commit_fetch_offsets`
  - [x] `TestAfkakClientIntegration.test_send_offset_request`
  - [x] `TestConsumerIntegration.test_consumer_commit_offsets`
  - [x] `TestConsumerIntegration.test_consumer_restart`
  - [x] `TestFailover.test_switch_leader`
  - [x] `TestAfkakProducerIntegration.test_produce_many_snappy`
  - [x] `TestAfkakProducerIntegration.test_produce_mixed`
- [x] Make tests pass on 1.1.1
  - [x] `test_produce_large_request`
- [x] Add 1.1.1 to the list of supported Kafkas

### CODE REVIEW

**Comment by twm** [comment]
It looks like we are going to see some intermittent failures in the integration tests on Kafka 1.1.1, but we can resolve those as they come up.

**Comment by twm** [comment]
I've seen this one a few times:

```
ERROR: afkak.test.test_consumer_integration.TestConsumerIntegration.test_consumer_restart
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/internet/utils.py", line 201, in runWithWarningsSuppressed
    reraise(exc_info[1], exc_info[2])
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/python/compat.py", line 463, in reraise
    raise exception.with_traceback(traceback)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/internet/utils.py", line 197, in runWithWarningsSuppressed
    result = f(*a, **kw)
  File ".../afkak/afkak/test/testutil.py", line 88, in wrapper
    return func(self)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/nose/twistedtools.py", line 169, in wrapper
    raise exc_type(exc_value).with_traceback(tb)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/nose/twistedtools.py", line 143, in errback
    failure.raiseException()
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/python/failure.py", line 385, in raiseException
    raise self.value.with_traceback(self.tb)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/internet/defer.py", line 1384, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/python/failure.py", line 408, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File ".../afkak/afkak/test/test_consumer_integration.py", line 191, in test_consumer_restart
    sent_messages = yield self.send_messages(self.partition, range(0, 100))
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/internet/defer.py", line 1384, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/python/failure.py", line 408, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File ".../afkak/afkak/test/test_consumer_integration.py", line 48, in send_messages
    resp, = yield self.client.send_produce_request([produce])
  File ".../afkak/.tox/py35-int-snappy/lib/python3.5/site-packages/twisted/internet/defer.py", line 1386, in _inlineCallbacks
    result = g.send(result)
  File ".../afkak/afkak/client.py", line 465, in send_produce_request
    returnValue(self._handle_responses(resps, fail_on_error, callback))
  File ".../afkak/afkak/client.py", line 566, in _handle_responses
    _check_error(resp)
  File ".../afkak/afkak/common.py", line 723, in _check_error
    raise error
afkak.common.UnknownTopicOrPartitionError
```

File: README.md [hunk 1]
```diff
@@ -24,13 +24,8 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md) for how to contribute.
...
 
 ### Kafka broker versions with which this version of Afkak is compatible:
-- 0.8.0 *
-- 0.8.1 *
-- 0.8.1.1 *
-- 0.8.2.1
 - 0.9.0.1
-
-\* See [Errata, below](#errata).
+- 1.1.1
```
Comment (rthille) [inline README.md]: Wat?   Only compatible with 1.1.1?

Comment (twm) [reply README.md]: I many run the tests against some older versions once coordinated consumer support is in, but for now this list only includes what we test against.

**Comment by twm** [comment]
Thank you for the review, @rthille!

File: README.md (hunk 1 above)
Comment (rthille) [reply README.md]: But that's not what it *says*: "### Kafka broker versions with which this version of Afkak is compatible:"

Comment (twm) [reply README.md]: Oh, good point... I will change the wording. My intent here is to drop support for brokers below 0.9.0.1.

**Pull request merged**